src/colmap/controllers/option_manager.h

```diff
@@ -122,16 +123,15 @@ class OptionManager {
   std::shared_ptr<std::string> image_path;
 
   std::shared_ptr<ImageReaderOptions> image_reader;
-  std::shared_ptr<SiftExtractionOptions> sift_extraction;
-
-  std::shared_ptr<SiftMatchingOptions> sift_matching;
+  std::shared_ptr<FeatureExtractionOptions> feature_extraction;
+  std::shared_ptr<FeatureMatchingOptions> feature_matching;
   std::shared_ptr<TwoViewGeometryOptions> two_view_geometry;
-  std::shared_ptr<ExhaustiveMatchingOptions> exhaustive_matching;
-  std::shared_ptr<SequentialMatchingOptions> sequential_matching;
-  std::shared_ptr<VocabTreeMatchingOptions> vocab_tree_matching;
-  std::shared_ptr<SpatialMatchingOptions> spatial_matching;
-  std::shared_ptr<TransitiveMatchingOptions> transitive_matching;
-  std::shared_ptr<ImagePairsMatchingOptions> image_pairs_matching;
+  std::shared_ptr<ExhaustivePairingOptions> exhaustive_matching;
```

Comment:
Should this be renamed to `exhaustive_pairing`? Also fine to do in another PR since this one is already huge.